modules/cluster/src/main/java/io/fluo/cluster/runner/YarnAppRunner.java

```diff
@@ -176,23 +223,23 @@ public void start() {
     try {
       // set twill run id zookeeper
       String twillId = controller.getRunId().toString();
-      CuratorUtil.putData(getCurator(), ZookeeperPath.YARN_TWILL_ID,
-          twillId.getBytes(StandardCharsets.UTF_8), CuratorUtil.NodeExistsPolicy.FAIL);
+      CuratorUtil.putData(getAppCurator(config), ZookeeperPath.YARN_TWILL_ID,
+          twillId.getBytes(StandardCharsets.UTF_8), CuratorUtil.NodeExistsPolicy.OVERWRITE);
```

Comment:
Why change from NodeExistsPolicy.FAIL to NodeExistsPolicy.OVERWRITE?  Not implying its wrong, just curious.